### ISSUE TITLE
Syncing problems on 18.03.0-ce-mac59

### ISSUE THREAD

**Issue by fosron** [EugenMayer/docker-sync#558]
### Error/Feature Requestion/Docs

Sync was working ok (ocassional problems but a `docker-sync sync` helped). Now with the update of Docker for Mac it stopped working again. I try `docker-sync stop && docker-sync start` but it would just sync some files generated on the container and not from the host (after a restart would just sync the same files ???). Tried restarting `Docker For Mac`, but it did not help.

Also when trying to run `docker-sync start` i get:

```
May take a long time (15min+) on the first run
          ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
          ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
          ok  Starting native_osx for sync applications-docker-sync
applications-docker-sync
unix:///tmp/supervisor.sock no such file
     success  Sync container started
     success  Starting Docker-Sync in the background
```

before restart of Docker for Mac i got this:

```ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
          ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
          ok  Starting native_osx for sync applications-docker-sync
applications-docker-sync
unison: ERROR (no such process)
unison: ERROR (no such process)
     success  Sync container started
```

Running `docker-sync sync`:

```docker-sync sync
          ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
          ok  Checking if a newer unison image is available
          ok  No newer image found - current image is up to date.
```

(no info in `docker-sync logs -f` generated)

### Docker Driver

Docker For Mac 18.03.0-ce-mac59

### Sync strategy

native_osx

### your docker-sync.yml

```version: "2"
options:
  verbose: false
syncs:
  applications-docker-sync: # name of the intermediary sync volume
    compose-dev-file-path: 'docker-compose.sync.yml' # docker-compose override file

    src: '${APPLICATION}' # host source directory
    sync_userid: 1000 # giving permissions to www-data user (as defined in nginx and php-fpm Dockerfiles)
    sync_strategy: '${DOCKER_SYNC_STRATEGY}' # for osx use 'native_osx', for windows use 'unison'

    sync_excludes: ['.git', 'node_modules', '.idea', 'remote_workspace.php.log', 'remote.php.log'] # ignored directories

    monit_enabled: true # not needed, this is the default

    # increase tolerance of high cpu if regular syncing uses a lot cpu
    monit_interval: 10
    monit_high_cpu_cycles: 6
```

### .env

```
APPLICATION=../
DOCKER_SYNC_STRATEGY=native_osx
```

### OS

macOS High Sierra 10.13.3 (17D47)

**Comment by EugenMayer** [comment]
Do not expect any fixes for this soonish - rather try to work on this yourself if you want to stick with mac59 - or simpley downgrade to mac42 which is the far best version with docker-sync (for now)

**Comment by fosron** [comment]
Ah damn, i see...

**Comment by mpbarlow** [comment]
But 18.03.0-ce-mac59 *is* stable, it's not an edge version

**Comment by EugenMayer** [comment]
i think this issue is actually related to https://github.com/EugenMayer/docker-sync/issues/553 and will be fixed by 0.5.7 using the new unison image

**Comment by mpbarlow** [comment]
https://docs.docker.com/docker-for-mac/release-notes/#stable-releases-of-2018

Looks like 59 was released two days ago, possibly a staggered roll out?

**Comment by EugenMayer** [comment]
Kind of odd, the stable box has mac58 offered just yesterday, my edge box has 59 - so idnk.

**Comment by barat** [comment]
@EugenMayer - stating that you support only stable is perfectly fine (knowing that as opensource project maintainer you have limited time resources) and I understand this. Maybe write such statement on Wiki/README.md ?
I'm using edge because I hope, that someday bind-mounting won't be close to useless, but because now docker-sync is (sort of) reliable I may switch back to stable :)

**Comment by EugenMayer** [comment]
@barat i truely have the same hopes, but for now, even the new NFS mount, is just not fast enough and i cannot help it.

State that we support stable only makes a lot of sense truly, but its for the wiki. Not sure where, do you have recommandation? Installation?

**Comment by barat** [comment]
IMO Installation, or separate "Requirements" where you can write about minimum Mac version, minimum Ram/CPU, Docker for Mac version XXX etc.

**Comment by EugenMayer** [comment]
@barat what about being bold and just introduce those changes? just help out :)

**Comment by EugenMayer** [comment]
really odd 
![image](https://user-images.githubusercontent.com/136934/38051329-3f8d5f7a-32ce-11e8-8a43-d9fd254854d3.png)

**Comment by EugenMayer** [comment]
very weired, on the client i was on mac58 - that was actually the edge channel, while 59 was the stable  one. Ok thats really crazy :D

**Comment by EugenMayer** [comment]
as expected it was related to https://github.com/EugenMayer/docker-sync/issues/553 and is fixed with 0.5.7 which will be release just right now
